File: Inválidos/interestelar.py
N = int(input())
[L, U] = [int(x) for x in input().split(' ')]

if L > U:
    temp = L
    L = U
    U = temp

Dados = [int(input()) for x in range(N)]

while True:
    try:
        [X, Y] = [int(x) for x in input().split(' ')]
        if X > Y:
            temp = X
            X = Y
            Y = temp
        
        DadosSelecionados = [Dados[x] for x in range(X, Y + 1)]
        Quantidade = len([x for x in DadosSelecionados if x >= L and x <= U])
        print(Quantidade)
    except:
        break

File: Inválidos/interestelar.c
#include <stdio.h>

int main(void) {
    int N, L, U, n;
    scanf("%d", &N);
    getchar();

    scanf("%d %d", &L, &U);
    getchar();

    if (L > U) {
        int temp = L;
        L = U;
        U = temp;
    }
    
    int Dados[N];
    for (n = 0; n < N; n++) {
        scanf("%d", &Dados[n]);
        getchar();
    }

    int X, Y, i;
    while (scanf("%d %d", &X, &Y) != EOF) {
        getchar();

        if (X > Y) {
            int temp = X;
            X = Y;
            Y = temp;
        }
        
        int Quantidade = 0;
        for (i = X; i <= Y; i++) {
            if (Dados[i] >= L)
                if (Dados[i] <= U)
                    Quantidade++;
        }
        printf("%d\n", Quantidade);
    }
    return 0;
}